tables/superdarn/radar.dat

```diff
@@ -62,8 +62,8 @@
 50   1 20190625 25000101 "Jiamusi East"            "National Space Science Center, CAS"  "hdw.dat.jme" "jme"
 51   1 20230626 25000101 "Longjing East"           "National Space Science Center, CAS"  "hdw.dat.lje" "lje"
 52   1 20230626 25000101 "Longjing West"           "National Space Science Center, CAS"  "hdw.dat.ljw" "ljw"
-53   1 20221101 25000101 "Siziwanqi East"          "National Space Science Center, CAS"  "hdw.dat.sze" "sze"
-54   1 20221101 25000101 "Siziwanqi West"          "National Space Science Center, CAS"  "hdw.dat.szw" "szw"
+53   1 20221101 25000101 "Siziwang East"          "National Space Science Center, CAS"  "hdw.dat.sze" "sze"
+54   1 20221101 25000101 "Siziwang West"          "National Space Science Center, CAS"  "hdw.dat.szw" "szw"
```

Comment:
Ideally the column alignment should be retained after shortening the radar names

Reply:
Good catch, I know there was something off when I did it before, not sure how I didn't see it.  I've got this and another set of corrections I missed before coming up.